### PR TITLE
Changelog as release description

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: .github/workflows/bindings-nodejs-publish.yml [hunk 1]
```diff
@@ -246,6 +246,11 @@ jobs:
...
-      - name: Upload prebuild to GitHub release
-        run: npx prebuild --upload-all ${{ secrets.GITHUB_TOKEN }} --tag-prefix iota-sdk-nodejs-v
-        working-directory: bindings/nodejs
+      - name: Upload prebuild to Github release
+        uses: softprops/action-gh-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/CHANGELOG.md
+          files: bindings/nodejs/prebuilds/@iota/*
+          tag_name: iota-sdk-nodejs-v${npm pkg get version --workspaces=false | tr -d \"}
```
Comment (thibault-martinez) [inline .github/workflows/bindings-nodejs-publish.yml]: not fully sure about that

Comment (kwek20) [reply .github/workflows/bindings-nodejs-publish.yml]: works for me

**Review by kwek20 — DISMISSED** [review]

File: .github/workflows/bindings-nodejs-publish.yml (hunk 1 above)
Comment (Thoralf-M) [reply .github/workflows/bindings-nodejs-publish.yml]: This is good stuff, copied from my working code :trollface: 
At least the `npm pkg get version --workspaces=false | tr -d \"`
Don't we need double curly brackets ${{ }} in the workflow?

Comment (thibault-martinez) [reply .github/workflows/bindings-nodejs-publish.yml]: Yeah that's what I was referring to, I don't really know

Comment (Thoralf-M) [reply .github/workflows/bindings-nodejs-publish.yml]: Actually might need to change more https://github.com/softprops/action-gh-release/issues/360

File: .github/workflows/bindings-nodejs-publish.yml [hunk 2]
```diff
@@ -224,6 +224,11 @@ jobs:
           security -v list-keychains -s signing.keychain
           security find-identity -vp codesigning
 
+      - name: Prepare Github release
+        id: prepare_release
+        run: |
+          echo "tag_name=iota-sdk-nodejs-v$(npm pkg get version --workspaces=false | tr -d \")" >> "$GITHUB_OUTPUT"
```
Comment (thibault-martinez) [inline .github/workflows/bindings-nodejs-publish.yml:230]: Not sure if we can touch the name L239, maybe we could reuse this. But don't know how it related to the npm releases.

Comment (Thoralf-M) [reply .github/workflows/bindings-nodejs-publish.yml:230]: It reads the version from the package.json so should be all good, but not sure if's worth it to reuse

**Review by Thoralf-M — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Review by abdulmth — APPROVED** [review]

**Pull request merged**